generic3g/MAPL_Generic.F90

```diff
@@ -143,10 +145,20 @@ module mapl3g_Generic
       procedure :: gridcomp_connect_all
    end interface MAPL_ConnectAll
 
-
    interface MAPL_ResourceGet
-      procedure :: hconfig_get_string
+      module procedure :: mapl_resource_get_i4_gc
+      module procedure :: mapl_resource_get_i8_gc
+      module procedure :: mapl_resource_get_r4_gc
+      module procedure :: mapl_resource_get_r8_gc
+      module procedure :: mapl_resource_get_logical_gc
+      module procedure :: mapl_resource_get_i4seq_gc
+      module procedure :: mapl_resource_get_i8seq_gc
+      module procedure :: mapl_resource_get_r4seq_gc
+      module procedure :: mapl_resource_get_r8seq_gc
+      module procedure :: mapl_resource_get_logical_seq_gc
+      module procedure :: mapl_resource_get_string_gc
```

Comment:
Latest Fortran versions dropped the need for `module` in this context.   And ... we don't need  the `mapl_` prefix on procedure names except for the public interface.
```suggestion
      procedure :: resource_get_i4_gc
      procedure :: resource_get_i8_gc
      procedure :: resource_get_r4_gc
      procedure :: resource_get_r8_gc
      procedure :: resource_get_logical_gc
      procedure :: resource_get_i4seq_gc
      procedure :: resource_get_i8seq_gc
      procedure :: resource_get_r4seq_gc
      procedure :: resource_get_r8seq_gc
      procedure :: resource_get_logical_seq_gc
      procedure :: resource_get_string_gc
```